qa/build/scala.Dockerfile

```
@@ -47,7 +47,17 @@
-    n 20
+    n 23
+
+# Upgrage yarn to latest version
+# https://yarnpkg.com/corepack
+RUN npm install -g corepack
+RUN corepack enable
+RUN yarn set version latest
+RUN yarn install
+
+# Install global Node/Yarn dependencies
+RUN npm install -g junit-report-merger
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)